benchmarks/test_calllater.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for scheduled events."""
+
+import pytest
+
+from twisted.internet.base import ReactorBase
+
+
+class Reactor(ReactorBase):
+    """
+    Implement minimal methods a subclass needs.
+    """
```

Comment:
I think that this need more documentation.

My initial question was: 
why not use [t.i.task.Clock](https://docs.twistedmatrix.com/en/stable/api/twisted.internet.task.Clock.html) ?

```suggestion
class ReactorWithRiggedScheduler(ReactorBase):
    """
    Allows controlling the passed time while testing the C{ReactorBase}.
    """
```

Reply:
It's not `Clock` because that would be benchmarking `Clock`, which is a completely different implementation.